pkgs/data/icons/whitesur-cursors/default.nix

```diff
@@ -1,22 +1,23 @@
-{ lib
-, stdenvNoCC
-, fetchFromGitHub
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
 }:
 
 stdenvNoCC.mkDerivation {
   pname = "whitesur-cursors";
-  version = "unstable-2022-06-17";
+  version = "0-unstable-2022-06-17";
 
   src = fetchFromGitHub {
     owner = "vinceliuice";
     repo = "WhiteSur-cursors";
     rev = "5c94e8c22de067282f4cf6d782afd7b75cdd08c8";
-    sha256 = "sha256-CFse0XZzJu+PWDcqmvIXvue+3cKX47oavZU9HYRDAg0=";
+    hash = "sha256-CFse0XZzJu+PWDcqmvIXvue+3cKX47oavZU9HYRDAg0=";
   };
 
   installPhase = ''
     runHook preInstall
-    install -dm 755 $out/share/icons/WhiteSur-cursors
+    mkdir -p $out/share/icons/WhiteSur-cursors
```

Comment:
Why is this?

Reply:
We may be more inclined to write `install` (instead of `mkdir` and `cp`) at present, although I may not be able to find the tracking issue now

Reply:
Note that this is not `install -D` but `install -d` which is AFAIK just directory creation.
Sadly, `install -D` doesn't support moving nested directories, that's why I had to use `cp`.

Reply:
So since there is nothing wrong with it, why do you want to change it?

Reply:
I mean, since it works normally and looks normally, there is no need to change it. What if another person comes and changes it back to `install` tomorrow?

Reply:
Okay, I think we won't really agree on this, so I'll revert this change, it's not that important.